<?xml version="1.0"?>
<launch>
  <param name = "acceleration_x_m"     value = "-9.8"/>
  <param name = "acceleration_y_m"     value = "-9.8"/>
  <param name = "acceleration_z_m"     value = "-9.8"/>

  <param name = "acceleration_x_b"     value = "-0.345"/>
  <param name = "acceleration_y_b"     value = "0.653"/>
  <param name = "acceleration_z_b"     value = "1.01"/>


  <param name = "gyro_x_offset"     value = "-5.647374045801527"/>
  <param name = "gyro_y_offset"     value = "2.040763358778626"/>
  <param name = "gyro_z_offset"     value = "-1.4516030534351145"/>

  <node name="mpu_node" pkg="mpu6050" type="imu_node" output="screen"/>

</launch>
